code/src/reducers/game.js

```diff
@@ -0,0 +1,78 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const game = createSlice({
+	name: 'game',
+	initialState: {
+		username: null,
+		started: false,
+		position: {},
+		type: '',
+		direction: '',
```

Comment:
I could be wrong but since the type and direction are sent as props to continueGame, so I think these can be removed from the initalState, unless they're used somewhere else that I've missed!